src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return calculator;
```

Comment:
Use 'return this;' instead of 'return calculator;' to maintain the correct context. Please check all methods